positive_charge/App.jsx

```diff
@@ -23,4 +25,6 @@ class App extends React.Component {
     }
 }
 
-ReactDOM.render(<App/>, document.getElementById('App'));
+const container = document.getElementById('app')
+const root = createRoot(container)
+root.render(<App />);
```

Comment:
Is this the new thing with react 18.0 ? 
I saw the error in the console but didn't look into yet.